internal/witcli/dir.go

```diff
@@ -0,0 +1,28 @@
+package witcli
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+)
+
+// FindOrCreateDir attempts to load file information for the provided path and
+// the provided path does not already exist, it attempts to create the directory
+// and then return the file information for it.
+func FindOrCreateDir(path string) (fs.FileInfo, error) {
+	info, err := os.Stat(path)
+	if os.IsNotExist(err) {
```

Comment:
**suggest**: use `if errors.Is(err, fs.ErrNotExist)`

Reply:
Good call, I'd honestly not realized that this wasn't a direct translation, but the docs state as much, thanks!